src/Devices/Artemis.Plugins.Devices.Wooting/Services/LedMapping.cs

```diff
@@ -0,0 +1,152 @@
+        [LedId.Keyboard_BracketRight] = 48,
+        [LedId.Keyboard_Backslash] = 49,
+        [LedId.Keyboard_NonUsTilde] = 50,
+        [LedId.Keyboard_SemicolonAndColon] = 51,
+        [LedId.Keyboard_ApostropheAndDoubleQuote] = 52,
+        [LedId.Keyboard_GraveAccentAndTilde] = 53,
+        [LedId.Keyboard_CommaAndLessThan] = 54,
+        [LedId.Keyboard_PeriodAndBiggerThan] = 55,
+        [LedId.Keyboard_SlashAndQuestionMark] = 56,
+
+        [LedId.Keyboard_CapsLock] = 57,
+
+        [LedId.Keyboard_F1] = 58,
+        [LedId.Keyboard_F2] = 59,
+        [LedId.Keyboard_F3] = 60,
+        [LedId.Keyboard_F4] = 61,
+        [LedId.Keyboard_F5] = 62,
+        [LedId.Keyboard_F6] = 63,
+        [LedId.Keyboard_F7] = 64,
+        [LedId.Keyboard_F8] = 65,
+        [LedId.Keyboard_F9] = 66,
+        [LedId.Keyboard_F10] = 67,
+        [LedId.Keyboard_F11] = 68,
+        [LedId.Keyboard_F12] = 69,
+
+        [LedId.Keyboard_PrintScreen] = 70,
+        [LedId.Keyboard_ScrollLock] = 71,
+        [LedId.Keyboard_PauseBreak] = 72,
+        [LedId.Keyboard_Insert] = 73,
+        [LedId.Keyboard_Home] = 74,
+        [LedId.Keyboard_PageUp] = 75,
+        [LedId.Keyboard_Delete] = 76,
+        [LedId.Keyboard_End] = 77,
+        [LedId.Keyboard_PageDown] = 78,
+
+        [LedId.Keyboard_ArrowRight] = 79,
+        [LedId.Keyboard_ArrowLeft] = 80,
+        [LedId.Keyboard_ArrowDown] = 81,
+        [LedId.Keyboard_ArrowUp] = 82,
+
+        [LedId.Keyboard_NumLock] = 83,
+        [LedId.Keyboard_NumSlash] = 84,
+        [LedId.Keyboard_NumAsterisk] = 85,
+        [LedId.Keyboard_NumMinus] = 86,
+        [LedId.Keyboard_NumPlus] = 87,
+        [LedId.Keyboard_NumEnter] = 88,
+        [LedId.Keyboard_Num1] = 89,
+        [LedId.Keyboard_Num2] = 90,
+        [LedId.Keyboard_Num3] = 91,
+        [LedId.Keyboard_Num4] = 92,
+        [LedId.Keyboard_Num5] = 93,
+        [LedId.Keyboard_Num6] = 94,
+        [LedId.Keyboard_Num7] = 95,
+        [LedId.Keyboard_Num8] = 96,
+        [LedId.Keyboard_Num9] = 97,
+        [LedId.Keyboard_Num0] = 98,
+        [LedId.Keyboard_NumPeriodAndDelete] = 99,
+
+        [LedId.Keyboard_NonUsBackslash] = 100,
+        [LedId.Keyboard_Application] = 101,
+
+        [LedId.Keyboard_LeftCtrl] = 224,
+        [LedId.Keyboard_LeftShift] = 225,
+        [LedId.Keyboard_LeftAlt] = 226,
+        [LedId.Keyboard_LeftGui] = 227,
+        [LedId.Keyboard_RightCtrl] = 228,
+        [LedId.Keyboard_RightShift] = 229,
+        [LedId.Keyboard_RightAlt] = 230,
+        [LedId.Keyboard_RightGui] = 231,
+
+        /**
+         * Thanks Simon for handing me this list
+         * RgbBrightnessUp = 0x401
+         * RgbBrightnessDown = 0x402
+         * SelectAnalogProfile1 = 0x403
+         * SelectAnalogProfile2 = 0x404
+         * SelectAnalogProfile3 = 0x405
+         * ModeKey = 0x408
+         * FnKey = 0x409
+         * DisableKey = 0x40A
+         * FnLayerLock = 0x40B
+         * FnKey2 = 0x40C= 
+         */
```

Comment:
```suggestion
        /*
         Thanks Simon for handing me this list
         RgbBrightnessUp = 0x401
         RgbBrightnessDown = 0x402
         SelectAnalogProfile1 = 0x403
         SelectAnalogProfile2 = 0x404
         SelectAnalogProfile3 = 0x405
         ModeKey = 0x408
         FnKey = 0x409
         DisableKey = 0x40A
         FnLayerLock = 0x40B
         FnKey2 = 0x40C= 
         */
```

What you have above is a XML comment, not a block comment ^^